src/tests/sqlsmith/src/sql_gen/query.rs

```diff
@@ -204,7 +220,11 @@ impl<'a, R: Rng> SqlGenerator<'a, R> {
                 let select_num = self.rng.gen_range(1..=5);
                 for _ in 0..select_num {
                     let ty = self.gen_data_type();
-                    let expr = self.gen_expr(&ty);
+                    let expr = if self.rng.gen_bool(0.8) {
+                        self.gen_agg_func(&ty)
```

Comment:
Aggregate functions can't occur at the same time as other expressions
for example
```sql
mysql> select max(a), b from test;
ERROR 1105 (HY000): SemanticError. Code: 1065, Text = column "b" must appear in the GROUP BY clause or be used in an aggregate function.
```